### DISCUSSION TITLE
Need Some useful steps for Production [V5]

### DISCUSSION THREAD

**Discussion by pk4all** [adonisjs/core#2492] [Help]
Hi, 
@thetutlage and @RomainLanz  
I need some help, I have go to live with my app Need some useful Instruction from your side for APIs fast response.
I have used EC2 large Instance for deployment.

Thanks.

**Comment by McSneaky** [comment]
That's really general question.. Is there something specific you are looking for?

Easiest way to increase API responses is to reduce computing and all kinds of IO and move server closer to customers to reduce latency
